Automattic/wp-calypso
# Site Settings: Add Disconnect Confirmation Page

Based on and supersedes @AnnaMag's #18039.

Add a route ` /settings/disconnect-site/confirm/:site` that displays the following:

![image](https://user-images.githubusercontent.com/96308/31520871-b0c8163a-afa7-11e7-8a38-d22a4a16086e.png)

Note that nothing currently points to that route; however, it will be the final step of the Disconnect JP Survey which I'll be adding with #18749.

To test:
* Verify that 'Stay Connected' takes you to `settings/manage-connection/:site`.
* Verify that 'Disconnect' actually disconnects your JP site, and redirects you to 'Stats'
* Try the route for a non-JP site; you should be redirected to `/settings/general`.
